variables.tf

```diff
@@ -57,3 +82,9 @@ variable "skip_cos_iam_authorization_policy" {
   default     = false
   description = "Set to true to skip the creation of an IAM authorization policy that permits the SCC instance created by this module to write access to the provided COS instance"
 }
+
+variable "attach_wp" {
+  type        = bool
+  default     = false
+  description = "Set to true to attach a Workload Protection instance to an SCC instance"
+}
```

Comment:
no need for this variable - if the user sets a value for `wp_instance_crn` then we know they want to attach one. We will add a count around the resource using that